src/parser.ts

```diff
@@ -158,41 +160,54 @@ export function parseLogStream(logStream: QuasiSubject<string>): QuasiObservable
           meta,
           rowIndex: progress.index++,
           row,
+          info: parseJsonInfo(row.Info),
```

Comment:
Can we do `row[FlightLogHeader.Info]` here instead? Would prefer to use the enum to address keys in the object rather than using those keys directly.

Reply:
Not sure what enum you're referring to. This is an additional field that represents the parsed data from the info column and not the info itself, which is still retained in the row.

Reply:
Oh, one other, maybe non-obvious thought here. The `FlightLog.Ifo` type is a string. I didn't want to change that in order to retain backward compatibility